src/modules/cmdpal/Microsoft.CmdPal.UI.ViewModels/CommandSettingsViewModel.cs

```diff
@@ -27,4 +34,27 @@ public void InitializeProperties()
             SettingsPage.InitializeProperties();
         }
     }
+
+    public void SafeInitializeProperties()
+    {
+        try
+        {
+            UnsafeInitializeProperties();
+        }
+        catch (Exception ex)
+        {
+            Logger.LogError($"Failed to load settings page", ex: ex);
+        }
+
+        Initialized = true;
```

Comment:
Are we sure we want to set this to true if there was an exception?

Reply:
Maybe in the future we'll care more about what the exception was and handle it differently depending.